kikuchipy/signals/ebsd.py

```diff
@@ -1606,9 +1606,8 @@ def hough_indexing(
         Parameters
         ----------
         phase_list
-            List of phases. The list can only contain one face-centered
-            cubic (FCC) phase, one body-centered cubic (BCC) phase or
-            both types.
+            List of phases. The list must correspond to the phase list
+            in the passed.
```

Comment:
```suggestion
            List of phases. The list must correspond to the passed phase list argument.
```

Reply:
Our [code style](https://kikuchipy.org/en/stable/dev/code_style.html) says:

> Comment lines should preferably be limited to 72 characters.

I consider docstrings to be comment lines. I'll update this sentence in the docs to make this clearer.